server/groundstation.py

```diff
@@ -9,7 +10,7 @@
 
 
 class GroundStation:
-    def __init__(self, socket):
+    def __init__(self, s):
```

Comment:
IMO it's not immediately understood `s` is a socket so it makes it a little less readable. I don't think it's very important though, so whatever. Did the linter suggest this?

Reply:
Nope - the issue is a confusion between the `socket` parameter and the `socket` library being imported. Didn't want to handle any of this, so I renamed it.